supporting_scripts/analysis-of-endpoint-connections/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+group 'de.tum.in.www1.artemis'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'com.thoughtworks.qdox:qdox:2.0-M9'
```

Comment:
we already include these dependencies in the "normal" build. Can we not reuse those definitions?

Reply:
```suggestion
    implementation 'com.thoughtworks.qdox:qdox:2.1.0'
```

Reply:
Yes, it is possible to reuse those definitions.